### PR TITLE
Add ktlint formatter

### PR DESCRIPTION
**Pull request by shurizzle**

### CODE REVIEW

File: lua/conform/formatters/ktlint.lua [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://ktlint.github.io/",
+    description = "An anti-bikeshedding Kotlin linter with built-in formatter.",
+  },
+  command = "ktlint",
+  args = { "--format", "--stdin", "**/*.kt", "**/*.kts" },
```
Comment (stevearc) [inline lua/conform/formatters/ktlint.lua:8]: do you need the globs? Based on [the docs](https://pinterest.github.io/ktlint/1.0.1/install/cli/#stdin-stdout) it seems like those aren't necessary if you're passing `--stdin`

File: lua/conform/formatters/ktlint.lua [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  stdin = true,
```
Comment (stevearc) [inline lua/conform/formatters/ktlint.lua:9]: this is the default and can be omitted

**Review by stevearc — CHANGES_REQUESTED** [review]

File: lua/conform/formatters/ktlint.lua (hunk 1 above)
Comment (shurizzle) [reply lua/conform/formatters/ktlint.lua:8]: Without paths it prints out
`18:40:35.484 [main] INFO com.pinterest.ktlint.cli.internal.KtlintCommandLine - Enable default patterns [**/*.kt, **/*.kts]`
1) I don't know how to exclude stderr from output
2) Those paths are enabled automatically

Any hints?

File: lua/conform/formatters/ktlint.lua (hunk 2 above)
Comment (shurizzle) [reply lua/conform/formatters/ktlint.lua:9]: Usually I like to be explicit, do you want me to delete this line?

File: lua/conform/formatters/ktlint.lua (hunk 1 above)
Comment (shurizzle) [reply lua/conform/formatters/ktlint.lua:8]: I saw null-ls.nvim and they did the same

Comment (stevearc) [reply lua/conform/formatters/ktlint.lua:8]: From the same docs:
`Logging output printed to stdout can be suppressed by setting --log-level=none`

Not sure why that _wouldn't_ be enabled by default when using stdin/stdout, but it seems like a good place to start

**Comment by stevearc** [comment]
Thanks for the PR!

**Pull request merged**